spec.html

```diff
@@ -12759,7 +12759,13 @@ <h1>
               1. If _Desc_ has a [[Enumerable]] field, let _enumerable_ be _Desc_.[[Enumerable]]; else let _enumerable_ be _current_.[[Enumerable]].
               1. Replace the property named _P_ of object _O_ with a data property whose [[Configurable]] and [[Enumerable]] attributes are set to _configurable_ and _enumerable_, respectively, and whose [[Value]] and [[Writable]] attributes are set to the value of the corresponding field in _Desc_ if _Desc_ has that field, or to the attribute's <emu-xref href="#table-object-property-attributes">default value</emu-xref> otherwise.
             1. Else,
-              1. For each field of _Desc_, set the corresponding attribute of the property named _P_ of object _O_ to the value of the field.
+              1. Let _prop_ be the property of object _O_ whose key is _P_.
+              1. If _Desc_ has a [[Configurable]] field, set the [[Configurable]] attribute of _prop_ to _Desc_.[[Configurable]].
```

Comment:
```suggestion
              1. If _Desc_ has a [[Configurable]] field, set _prop_'s [[Configurable]] attribute to _Desc_.[[Configurable]].
```
and similarly below. Based on the precedent in [OrdinaryGetOwnProperty](https://tc39.es/ecma262/#sec-ordinarygetownproperty)